test/core/client_channel/lb_policy/xds_override_host_test.cc

```diff
@@ -37,6 +34,28 @@ class XdsOverrideHostTest : public LoadBalancingPolicyTest {
          Json::Object{{"childPolicy", Json::Array{{child_policy_config}}}}}}});
   }
 
+  void SetUp() override {
+    grpc_core::SetEnv("GRPC_EXPERIMENTAL_XDS_ENABLE_HOST_OVERRIDE", "TRUE");
+    grpc_init();
```

Comment:
I think you can move `grpc_init()` and `grpc_shutdown()` back down to `main()`.  Instead, just add `grpc_core::CoreConfiguration::Reset()` here (in addition to the one in `TearDown()`), just to make sure that the env var takes effect.